website/webpack.config.js

```diff
@@ -39,24 +39,60 @@ module.exports = {
         test: /\.css$/,
         use: [{ loader: 'style-loader' }, { loader: 'css-loader' }],
       },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              publicPath: 'assets',
+              context: __dirname,
+              outputPath(_url, resourcePath, context) {
+                return path.relative(context, resourcePath);
+              },
+              name: '[name].[ext]',
+            },
+          },
+        ],
+      },
+      {
+        test: /\.webmanifest$/,
```

Comment:
The images don't use the hash for the name, so what is the purpose of the loaders for the manifest? It doesn't work with a plain copy? Nothing seems to be dynamic inside the manifest.

Reply:
That was the first solution but we then changed that with @yannickcr.

I updated to use `CopyWebpackPlugin` which seems to be easier to use in this case.